unlock-app/src/components/content/VerificationContent.tsx

```diff
@@ -42,7 +64,11 @@ export const VerificationContent: React.FC<unknown> = () => {
           addLock,
         }}
       >
-        <VerificationStatus data={data} sig={sig} />
+        <VerificationStatus
+          rawData={query.data as string}
```

Comment:
Do we need both the raw and not raw data?

Reply:
Yes. I've moved validation and parsing above. For checking signature, you need the rawData.